README.md

```diff
@@ -1,2 +1,68 @@
-# yast-rake-ci
-Extension Rake tasks for the yast-rake gem
+
+YaST CI Tasks
+=============
+
+[![Build Status](http://ci.opensuse.org/buildStatus/icon?job=yast-rake-ci-master)](http://ci.opensuse.org/view/Yast/job/yast-rake-ci-master/)
+[![Code Climate](https://codeclimate.com/github/yast/yast-yast-rake-ci/badges/gpa.svg)](https://codeclimate.com/github/yast/yast-rake-ci)
+
+
+Description
+============
+
+This package provides extensions for the [YaST Rake tasks](https://github.com/yast/yast-rake).
+
+These tasks are needed only during development and in Jenkins build, this package
+in not needed at run-time or in OBS for building the YaST packages.
+
+The package installs all needed packages via RPM dependencies.
+
```

Comment:
Honestly, I'm still missing information *why* this package exists and *what* it provides. All the info is there, but not in a human-readable form :) Please describe the *purpose* at the very top.